proxy/src/main/java/com/wavefront/agent/logsharvesting/LogsIngester.java

```diff
@@ -64,13 +64,21 @@ public LogsIngester(PointHandler pointHandler, Supplier<LogsIngestionConfig> log
     this.flushProcessor = new FlushProcessor(sent, currentMillis);
 
     // Set up user specified metric harvesting.
-    LogsIngestionConfig config = logsIngestionConfigManager.getConfig();
     this.pointHandler = pointHandler;
 
     // Continually flush user metrics to Wavefront.
     this.metricsReporter = new MetricsReporter(
         evictingMetricsRegistry.metricsRegistry(), flushProcessor, "FilebeatMetricsReporter", pointHandler, prefix);
-    this.metricsReporter.start(config.aggregationIntervalSeconds, TimeUnit.SECONDS);
+  }
+
+  public void start() {
+    this.metricsReporter.start(
```

Comment:
This is obviously not critical, but it would be nice to print a message as it begins reading from stdin to let the user know it's ready to accept data